x/auth/client/cli/query.go

```diff
@@ -210,28 +215,99 @@ $ %s query txs --%s 'message.sender=cosmos1...&message.action=withdraw_delegator
 // QueryTxCmd implements the default command for a tx query.
 func QueryTxCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "tx [hash]",
-		Short: "Query for a transaction by hash in a committed block",
-		Args:  cobra.ExactArgs(1),
+		Use:   "tx [hash|acc_seq|signature] --type=[hash|acc_seq|signature]",
+		Short: "Query for a transaction by hash, addr++seq combination or signature in a committed block",
+		Long: strings.TrimSpace(fmt.Sprintf(`
+Example:
+$ %s query tx <hash>
+$ %s query tx --%s=%s <addr>:<sequence>
+$ %s query tx --%s=%s <sig1_base64,sig2_base64...>
+`, version.AppName, version.AppName, flagType, typeAccSeq, version.AppName, flagType, typeSig)),
```

Comment:
```suggestion
`, version.AppName, 
version.AppName, flagType, typeAccSeq, 
version.AppName, flagType, typeSig)),
```

Reply:
let's make the arguments more readable by breaking them in the same way as we break the new lines in the formatting string.